src/api/telemetry/TelemetryCollection.js

```diff
@@ -70,7 +70,6 @@ export class TelemetryCollection extends EventEmitter {
         this._watchBounds();
         this._watchTimeSystem();
 
-        this._requestHistoricalTelemetry();
```

Comment:
Hrm, calling `this._requestHistoricalTelemetry();` here makes it really clear that historical telemetry is fetched on load. Now we are relying on historical telemetry being fetched because the time system is also set on load. It's no longer obvious to a developer how historical telemetry is gets fetched on load.

Here is an alternative proposal. We have two functions `_setTimeSystem()` and `_setTimeSystemAndFetchData()`. Literally all that the second one does is call first and then calls `reset()`. The first can be called from `load`, the second from the time system change listener.

Reply:
I like it. I love it.